src/hssmatrix.jl

```diff
@@ -138,13 +138,13 @@ function _getidx(hssA::HssMatrix, i::Int, j::Int)
       else
         U1 = _getindex_colgenerator(hssA.A11, i)
         V2 = _getindex_rowgenerator(hssA.A22, j-n1)
-        return dot(U1*hssA.B12, V2)
+        return U1*hssA.B12*V2'
```

Comment:
any reason for changing this? could you check which of the two is faster?

Reply:
These lines are not equivalent for complex valued object. 
The initial version result in a bug for the random element access.  
I just notice that I never added the test for it. It will be done in a new commit. 
Yet, I will check if there is performance changes.